controller/control.go

```diff
@@ -935,12 +936,14 @@ func (c *Controller) Start(addresses ...string) error {
 // on the app.
 func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 	c.Lock()
-	if c.ReadOnly == true {
+	if c.ReadOnly && !c.logged {
```

Comment:
We shouldn't skip this entire block, only logging needs to be disabled.

Reply:
good catch